.github/workflows/IJ.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IU-2019.3, IU-2020.1, IU-2020.2, IU-2020.3, IU-2021.1, IU-2021.2, IU-2021.3]
+        IJ: [IU-2021.1, IU-2021.2, IU-2021.3]
```

Comment:
Why those versions are removed ?

Reply:
I was taking openshift plugin as example https://github.com/redhat-developer/intellij-openshift-connector/blob/main/.github/workflows/validate_IJ_versions.yml#L12

Reply:
Yes but IJ OpenShift has a minimum IJ version requirement that is different from IJ Knative

Reply:
I put them back but needed to change to some more specific versions like 2019.3.1 and so on... because 2019.3 have been deleted from the available build versions file. The link can be found here https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
I omitted 2019.3.2 and 2019.3.3 and the same for 2020.x. Not sure if we want to run the workflow for everything or that's enough